tools/Resources.py

```diff
@@ -63,6 +64,7 @@ def get_virtualmachines(self, target, token, parentid):
     def get_vmfolders(self, target, token):
         return self.get_resources(target, token, parentid=None, resourcekind="VMFolder")
 
+    #not recommended
     def get_latest_stat(self, target, token, uuid, key):
         url = "https://" + target + "/suite-api/api/resources/" + uuid + "/stats/latest"
         headers = {
```

Comment:
that can be removed. There is no need for it anymore.

Reply:
I mean it is working and there for an easy single stat get. Not sure if we should throw it to the trash yet. However, I am going to remove `self` to make it callable.